### PR TITLE
1 connect wallet

### LINKED ISSUE #1: connect-wallet
connect wallet button

### PR DESCRIPTION
**Pull request by dousdogukan**
for #1 

- implement CButton
- implement CConnectWallet button 
- add menu to connect wallet button

### CODE REVIEW

File: src/Components/CButton/index.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+export type Variant = 'simple' | 'bordered';
+interface CButtonProps {
```
Comment (ceydae) [inline src/Components/CButton/index.tsx:4]: add new line here

File: src/Components/CConnectWallet/index.tsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
+import { useState } from 'react';
+
```
Comment (ceydae) [inline src/Components/CConnectWallet/index.tsx:2]: remove new line here

File: src/Components/CConnectWallet/index.tsx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+import { Menu, MenuButton, MenuItem, MenuItems, Transition } from '@headlessui/react';
+
+import CButton from '../CButton';
+import shortenAddress from '../../utils/shortenAddress';
+import copyToClipboard from '../../utils/copyToClipboard';
+
+import { ArrowRightStartOnRectangleIcon, SquareStackIcon } from './icons';
+
+const ConnectButton = () => {
+  const [address, setAddress] = useState<string>('');
```
Comment (ceydae) [inline src/Components/CConnectWallet/index.tsx:15]: remove the string generic

File: src/Components/CConnectWallet/index.tsx [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+  const handleConnect = async () => {
+    if (address) {
+      return;
+    }
+
+    freighterApi.isConnected().then((isConnected) => {
+      if (!isConnected) {
+        console.log('Freighter is not installed');
```
Comment (ceydae) [inline src/Components/CConnectWallet/index.tsx:24]: Wrong logic. Not connected does not mean that freghter is not installed. is it?

**Review by ruveyda-kisla — APPROVED** [review]

**Review by ceydae — APPROVED** [review]

**Pull request merged**